htdocs/bom/class/bom.class.php

```diff
@@ -580,7 +580,7 @@ public function delete(User $user, $notrigger = 1)
 	 *
 	 * @param	int		$fk_product				Id of product
 	 * @param	float	$qty					Quantity
-	 * @param	int		$qty_frozen				Frozen quantity
+	 * @param	float	$qty_frozen				Frozen quantity
```

Comment:
qty_frozen is not a qty but a status flag (0, 1, ...) to say if we want to have the qty modified depending on qty produced or a constant value.
So should be an int.